### DISCUSSION TITLE
Did scheduling break this week? (schedule:weekends triggering rebases during weekdays)

### DISCUSSION THREAD

**Discussion by MPV** [renovatebot/renovate#12907] [Request Help]
We've successfully been using `schedule:weekends` (https://docs.renovatebot.com/presets-schedule/#scheduleweekends) for months, but this week it's suddenly started rebasing our open PRs from master also during weekdays.

Did anything break or change?

**Comment by viceice** [comment]
https://docs.renovatebot.com/configuration-options/#updatenotscheduled

**Reply by MPV** [reply]
Nice, we'll give this a try!

**Comment by rarkins** [comment]
Seems unlikely, because our last changes to our main schedule coding was about a month ago: https://github.com/renovatebot/renovate/blob/main/lib/workers/branch/schedule.ts
